envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -64,8 +64,16 @@ class IncomingPermissionsProperties {
     var sourceIpAuthentication = SourceIpAuthenticationProperties()
 }
 
+typealias ClusterName = String
+
 class SourceIpAuthenticationProperties {
     var enabledForServices: List<String> = listOf()
+    var staticIpRanges: MutableMap<ClusterName, List<IpWithPrefix>> = mutableMapOf()
```

Comment:
Now it looks confusing. I know that property `enabledForServices` allow to specify services for which we want to do exact ip match but going through properties it looks like I have to specify for which services I want to enable this and then define ranges.

Reply:
How would you rename it? `exactIpAuthenticationForServices` and `staticIpRangePerService` ?

Reply:
@lukidzi suggested:

```
exactIpAuthentication:
  serviceNames:
staticIpAuthentication:
  ipRangeForService:
```

WDYT: @MarcinFalkowski @wookieJ @chemicL @pbetkier

Reply:
For me it's OK. I would only change ipRangeForService -> ipRangesForServices